packages/gulp-babili/src/index.js

```diff
@@ -0,0 +1,105 @@
+const babelCore = require("babel-core");
+const through2 = require("through2");
+const gutil = require("gulp-util");
+const applySourceMap = require("vinyl-sourcemaps-apply");
+const babiliPreset = require("babel-preset-babili");
+
+const {PluginError} = gutil;
+
+const NAME = "gulp-babili";
+
+module.exports = gulpBabili;
+
+function gulpBabili(_opts = {}) {
```

Comment:
will change this and take 2 arguments instead for passing in custom babel, babili and comments